quinn/transformations.py

```diff
@@ -87,28 +88,204 @@ def to_snake_case(s: str) -> str:
     return s.lower().replace(" ", "_")
 
 
-def sort_columns(df: DataFrame, sort_order: str) -> DataFrame:
+def sort_columns(
+    df: DataFrame, sort_order: str, sort_nested: bool = False
+) -> DataFrame:
     """This function sorts the columns of a given DataFrame based on a given sort
     order. The ``sort_order`` parameter can either be ``asc`` or ``desc``, which correspond to
     ascending and descending order, respectively. If any other value is provided for
     the ``sort_order`` parameter, a ``ValueError`` will be raised.
 
     :param df: A DataFrame
-    :type df: pandas.DataFrame
```

Comment:
😱 haha, good catch

Reply:
@MrPowers If you remember, I used ChatGPT for generating of docstrings, so such errors are expected...